python/vyos/configverify.py

```diff
@@ -36,8 +36,13 @@ def verify_mtu(config):
         mtu = int(config['mtu'])
 
         tmp = Interface(config['ifname'])
-        min_mtu = tmp.get_min_mtu()
-        max_mtu = tmp.get_max_mtu()
+        # Not all interfaces support min/max MTU
+        # https://vyos.dev/T5011
+        min_mtu = 68
+        max_mtu = 65536
+        if all([bool(min_mtu), bool(max_mtu), bool(mtu)]):
```

Comment:
Would this not always return true and call the functions that throw an exception?

Reply:
Maybe better to just wrap the calls in try/catch?

```
try:
    min_mtu = tmp.get_min_mtu()
    max_mtu = tmp.get_max_mtu()
except: # Fallback to defaults
    min_mtu = 68
    max_mtu = 65536
```

Reply:
I am for any option :) I wanted something new than "try/except"
```
>>> all([True, True, True])
True
>>> 
>>> all([True, True, 0])
False
>>> 

>>> all([True, True, False])
False
>>> 
```
Do you think it can cause a wrong True exception?